tests/Unit/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/Test_BjorhusImpl.cpp

```diff
@@ -148,10 +150,52 @@ void test_constraint_preserving_bjorhus_v_psi(
 }
 }  // namespace
 
+// Python tests
+namespace {
+template <size_t VolumeDim>
+tnsr::aa<DataVector, VolumeDim, Frame::Inertial> wrapper_func_v_psi(
+    const tnsr::I<DataVector, VolumeDim, Frame::Inertial>&
+        interface_normal_vector,
+    const tnsr::iaa<DataVector, VolumeDim, Frame::Inertial>&
+        three_index_constraint,
+    const tnsr::aa<DataVector, VolumeDim, Frame::Inertial>& char_proj_dt_v_psi,
+    const tnsr::a<DataVector, 3, Frame::Inertial>& char_speeds) {
+  std::array<DataVector, 4> char_speed_array{
+      {get<0>(char_speeds), get<1>(char_speeds), get<2>(char_speeds),
+       get<3>(char_speeds)}};
+  auto dt_v_psi =
+      make_with_value<tnsr::aa<DataVector, VolumeDim, Frame::Inertial>>(
+          get<0>(interface_normal_vector), 0.);
+  GeneralizedHarmonic::BoundaryConditions::detail::
+      set_dt_v_psi_constraint_preserving<VolumeDim, DataVector>(
+          make_not_null(&dt_v_psi), interface_normal_vector,
+          three_index_constraint, char_proj_dt_v_psi, char_speed_array);
+  return dt_v_psi;
+}
+
+template <size_t VolumeDim>
+void test_set_dt_v_psi_constraint_preserving(
+    const size_t grid_size_each_dimension) noexcept {
+  pypp::check_with_random_values<1>(
+      &wrapper_func_v_psi<VolumeDim>,
+      "Evolution.Systems.GeneralizedHarmonic.BoundaryConditions.Bjorhus",
+      "set_dt_v_psi_constraint_preserving", {{{-1., 1.}}},
+      DataVector(grid_size_each_dimension));
+}
+}  // namespace
+
 SPECTRE_TEST_CASE("Unit.Evolution.Systems.GeneralizedHarmonic.BCBjorhus.VPsi",
                   "[Unit][Evolution]") {
+  pypp::SetupLocalPythonEnvironment local_python_env{""};
```

Comment:
In other pypp tests, we've preferred to initialize the `local_python_env` to the current directory, and then the `check_with_random_values` call can just provide `"Bjorhus"` for the module -- that might be better for consistency
This doesn't make much of a difference when there's exactly one python module for the test, but it'll make it a bit more convenient when there's multiple calls to `check_with_random_values`

Reply:
An exception to this preference is when the python module itself requires to import from elsewhere from the root directory (`tests/Unit/`). For e.g. <https://github.com/sxs-collaboration/spectre/blob/0d615ebbffeb5937a891b0731d5164a8d04e3f92/tests/Unit/PointwiseFunctions/AnalyticData/GrMhd/Test_MagnetizedFmDisk.cpp#L181>. While `Bjorhus.py` does not yet make such imports, other PRs will soon add functions to the same file that will need to. This is why I set up the python environment at root in this test, as it would anyway have to be changed when, for e.g. #3000 gets merged.

Reply:
gotcha, makes sense, thanks!